src/main/kotlin/com/example/iot_management_device/model/Device.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.iot_management_device.model
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.hibernate.annotations.JdbcTypeCode
+import org.hibernate.type.SqlTypes
+
+@Entity
+@Table(name = "devices")
+data class Device(
```

Comment:
Why relational structure?

Reply:
In our on-boarding plan Mongo is set for week 4 so I decided to start from relational db and then migrate to non-relational